src/Block/MediaBlockService.php

```diff
@@ -184,7 +184,9 @@ public function validate(ErrorElement $errorElement, BlockInterface $block): voi
     }
 
     /**
-     * @return array<array-key, array-key>
```

Comment:
I would say that `@return array<int|string>` will avoid having two annotations.

Reply:
Fixed